app/jobs/load_organizations_job.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class LoadOrganizationsJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    load_organizations
+  end
+
+  private
+
+  def load_organizations
+    User.whitelisted_orgs.each do |org|
```

Comment:
`Organization.whitelisted` will be much more logical place for this list, since you now have the `Organisation` model